clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -191,6 +194,18 @@ invariant_checker_t::parallel_shard_memref(void *shard_data, const memref_t &mem
         shard->found_page_size_marker_ = true;
     }
 
+    if (memref.marker.type == TRACE_TYPE_MARKER &&
+        memref.marker.marker_type == TRACE_MARKER_TYPE_CHUNK_INSTR_COUNT) {
+        shard->chunk_instr_count_ = memref.marker.marker_value;
+    }
+    if (memref.marker.type == TRACE_TYPE_MARKER &&
```

Comment:
nit: should we add a comment here about this invariant check?

Reply:
Done.